test1

```diff
@@ -1,2 +1,3 @@
 some test file
```

Comment:
The way I see it, every life is a pile of good things and bad things. The good things don't always soften the bad things, but vice versa, the bad things don't always spoil the good things or make them unimportant. -11th Doctor

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Maintainable